timed/reports/tests/test_notify_reviewers_unverified.py

```diff
@@ -27,9 +35,16 @@ def test_notify_reviewers(db, mailoutbox):
     ReportFactory.create(date=date(2017, 7, 1), task=task_no_work,
                          verified_by=reviewer_no_work)
 
-    call_command('notify_reviewers_unverified')
+    call_command(
+        'notify_reviewers_unverified',
+        '--cc={0}'.format(cc),
```

Comment:
We still need to test what happens if the user leaves out the parameters. not just the parameters being empty

Reply:
I have added a test case for when the user leaves out the parameters.